vulnerabilities/improve_runner.py

```diff
@@ -36,11 +37,18 @@ class ImproveRunner:
     improver and parsing the returned Inferences into proper database fields
     """
 
-    def __init__(self, improver_class):
-        self.improver_class = improver_class
+    def __init__(self, improver_class=None, improver: Improver = None):
```

Comment:
May be keeping the class design and passing kwargs can work too?
```suggestion
    def __init__(self, improver_class, **kwargs):
    self.improver = improver_class(**kwargs)
```